doc/usecases/kdb/UC_store_config.md

```diff
@@ -0,0 +1,28 @@
+# Use Case: Storing application configuration
+
+## Summary
+
+- **Title:** Storing application configuration
```

Comment:
I don't think this use case is needed. "Modifying application configuration" is enough.

Reply:
> You could also store without loading first. I know it's not supported by kdbSet, but that doesn't matter. We're not documenting the API here, we're listing use cases.

If we don't support set without prior get, it shouldn't be present in the use cases.

Reply:
I disagree. The use case should define what we want. It is not the place to define limitations. The limitations of different implementations should be weighed in the decision stage. That is where we have a proper process to decide what limitations are acceptable. It's just that in this case, we are doing things backwards.

So in this specific case:
- Use case: We want to be able to store configuration data.
- Decision: Requiring a load before a store is acceptable. We can always load an empty config and then store the new config, if we don't have config yet. If we want to store a config regardless of whether one already exists, just load the existing one and discard it in memory. Then store the new config.
- Implementation: `kdbGet` & `kdbSet`